common-lib/src/cmr/common/generics.clj

```diff
@@ -117,3 +119,101 @@
                        (get (json/parse-string index-raw true) :SubConceptType "X")))))
           {}
           (latest-approved-documents)))
+;;TODO: Make sure comments have two semicolons on them
+
+(defn read-generic-doc-file
+  "Return the specific schema's documentation files given the schema keyword name and version number.
+   if the file cannot be read, return an empty string which will have no impact on the API document.
+   Parameters:
+   * file-name: [ingest | search]
+   * generic-keyword: [:grid | ...]
+   * generic-version: 0.0.1
+   Returns: string"
+  [file-name generic-keyword generic-version]
+  (try
+    (-> "schemas/%s/v%s/%s.md"
+        (format (name generic-keyword) generic-version (name file-name))
+        (io/resource)
+        (slurp))
+    (catch Exception e (str ""))))
```

Comment:
Would prefer at least a warning be logged.

Reply:
So this is to manage if there exists a markdown file for a given generic we include it in the documentation otherwise, we would be adding just an empty string.